### PR TITLE
Stacked component refactor

### PR DESCRIPTION
**Pull request by shanejann**
I'm refactoring the stacked component so it can be used for both our detail cards _as well as_ our summary cards. As a result, I'll also be removing the following:
- Card.ItemDetail
- Card.ItemLabel
- Card.ItemContainer

As these are all already covered within the StackedComponent.

### CODE REVIEW

File: src/screens/Transaction.js [hunk 1]
```diff
@@ -42,34 +43,34 @@ function TxDetailScreen({ hash }) {
           <Card.HeaderTitle>Transaction Summary</Card.HeaderTitle>
         </Card.Header>
         <Card.Content>
-          <Card.HorizontalContainer>
-            <Card.Column>
+          <Row>
```
Comment (shanejann) [inline src/screens/Transaction.js]: Not explicitly apart of my desired changes, but the Transaction.js page was busted without this and simply wasn't rendering any content.

File: src/components/address/AddressSummary.js [hunk 2]
```diff
@@ -44,39 +45,41 @@ export default function AddressSummary({ hash, received, spent, confirmed }) {
...
+                <StackedData
+                  label="Received"
+                  value={
+                    hnsValues(received) || (
+                        <SkeletonWrapper>
+                          <Skeleton />
+                        </SkeletonWrapper>
+                    )
+                  }
+                />
```
Comment (shanejann) [inline src/components/address/AddressSummary.js]: Let me know if we have any opinions on how I decided to layout this component when a complex value is passed to it. As you can see, I mainly just went with the "more indenting is better" approach

File: src/components/shared/StackedData.js [hunk 3]
```diff
@@ -3,34 +3,48 @@ import PropTypes from "prop-types";
...
-const StackedData = props => {
+const StackedBody = props => {
   return (
-    <Wrapper className="stacked">
-      <Label>{props.label}</Label>
-      <Value>
+    <>
+      <Label {...props}>{props.label}</Label>
+      <Value {...props}>
         {props.link ? <Link to={props.link}>{props.value}</Link> : props.value}
```
Comment (shanejann) [inline src/components/shared/StackedData.js]: I didn't think of this until I was finished refactoring, but we might be able to get rid of the `value` prop, and instead just use `children` here. Might be worth a second refactor?

File: src/screens/Transaction.js (hunk 1 above)
Comment (kilpatty) [reply src/screens/Transaction.js]: Yeah this is perfect, my bad for not including this in the other PR

File: src/components/shared/StackedData.js (hunk 3 above)
Comment (kilpatty) [reply src/components/shared/StackedData.js]: +1 Yes, I think that would be a nice DX improvement - only keeping Label as a prop.

**Pull request merged**